pennylane/qnn/cost.py

```diff
@@ -0,0 +1,119 @@
+"""
```

Comment:
Don't forget that the license needs to be placed at the top of the file! This can be copied from any other file present :)

Reply:
Done!